test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -643,6 +659,43 @@ def _assert_artifact_free(output, stray_artifacts):
             artifact, output.stdout
         ), f"Matched {artifact} in {output.stdout} while running {output.command}"
 
+def _test_sm_toolkit_and_ts_version(image, region):
```

Comment:
I see this is an SM only test. Just confirming - are we not adding labels for EC2 images?

Reply:
I dont think labels are needed for EC2 images as per what I spoke with toolkit team.

Reply:
The only potential gap here with EC2 images is if someone or some package installs TS outside of the "ARG" provided.

Reply:
Like I mentioned earlier, it will break at SageMaker image (as per our requirement) but will likely pass the EC2 image. I also checked with toolkit team and like I said, they don't worry about ec2 images/labels (atleast for the telemetry use case which is in scope for this requirement)